crates/biome_js_analyze/src/lint/nursery/use_sorted_classes/sort_config.rs

```diff
@@ -7,17 +7,33 @@
 
 use std::collections::HashMap;
 
+use bitvec::{order::Lsb0, vec::BitVec};
+
 /// A utility layer, containing its name and an ordered list of classes.
 pub struct UtilityLayer {
     pub name: &'static str,
     pub classes: &'static [&'static str],
 }
 
+pub struct Variant {
+    pub name: &'static str,
+    pub weight: BitVec<u8, Lsb0>,
+}
+
+/// This builds a bit vector ordered from Lsb to Msb in order to perform a simpler BitWise XOR later on
+/// Every variant has one bit set to 1 (Msb) and the others set to 0 and they have fixed size
```

Comment:
Not sure if this comment is that relevant, at least not as docs for the function - it's an implementation detail. FWIW, I intend to write about the Tailwind sorting algorithm in general so that could be used as reference for anyone trying to achieve a technical understanding of it.